src/analyserbeats.cpp

```diff
@@ -63,22 +63,22 @@ bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate, int totalSample
     }
 
     m_bPreferencesFixedTempo = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_FIXED_TEMPO_ASSUMPTION)).toInt());
+                                   m_pConfig->getValueString(
+                                       ConfigKey(BPM_CONFIG_KEY, BPM_FIXED_TEMPO_ASSUMPTION)).toInt());
     m_bPreferencesOffsetCorrection = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_FIXED_TEMPO_OFFSET_CORRECTION)).toInt());
+                                         m_pConfig->getValueString(
+                                                 ConfigKey(BPM_CONFIG_KEY, BPM_FIXED_TEMPO_OFFSET_CORRECTION)).toInt());
     m_bPreferencesReanalyzeOldBpm = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
+                                        m_pConfig->getValueString(
+                                            ConfigKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
```

Comment:
What is the alignment pattern here? 
```
   m_bPreferencesReanalyzeOldBpm = static_cast<bool>(
                                        m_pConfig->getValueString(
                                            ConfigKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
```

IMHO clang fails here as well: 
```
    m_bPreferencesReanalyzeOldBpm = static_cast<bool>(
            m_pConfig->getValueString(
                             ConfigKey(BPM_CONFIG_KEY,
                                       BPM_REANALYZE_WHEN_SETTINGS_CHANGE))
                    .toInt());
```

My eclipse: 
```
    m_bPreferencesReanalyzeOldBpm = static_cast<bool>(m_pConfig->getValueString(
            ConfigKey(BPM_CONFIG_KEY,
            BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
```

It looks like we reach the borders of auto formatter smartness here. 
I think developers help is required anyway: 
```
    ConfigKey reanalyzeKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE);
    m_bPreferencesReanalyzeOldBpm = 
            static_cast<bool>(m_pConfig->getValueString(reanalyzeKey).toInt());
```